lighthouse-core/audits/uses-rel-preconnect.js

```diff
@@ -113,17 +118,24 @@ class UsesRelPreconnectAudit extends Audit {
         return (record.startTime < firstRecord.startTime) ? record: firstRecord;
       });
 
-      const connectionTime =
-        firstRecordOfOrigin._timing.connectEnd - firstRecordOfOrigin._timing.dnsStart;
+      const securityOrigin = firstRecordOfOrigin.parsedURL.securityOrigin();
+
+      // Approximate the connection time with the duration of TCP (+potentially SSL) handshake
+      // DNS time can be large but can also be 0 if a commonly used origin that's cached, so make
+      // no assumption about DNS.
+      const additionalRtt = additionalRttByOrigin.get(securityOrigin) || 0;
+      let connectionTime = rtt + additionalRtt;
+      if (firstRecordOfOrigin.parsedURL.scheme === 'https') connectionTime = connectionTime * 2;
```

Comment:
comment on where the *2 comes from (is that the TLS mentioned above? Why *2?)

Reply:
done

Reply:
done!